index.js

```diff
@@ -2,50 +2,20 @@
 /*
   Initial code from https://github.com/gulpjs/gulp-util/blob/v3.0.6/lib/log.js
  */
-var chalk = require('chalk');
+var grey = require('chalk').grey;
```

Comment:
if we only need gray, maybe we should consider using [ansi-gray](https://www.npmjs.com/package/ansi-gray)?